account/views.py

```diff
@@ -0,0 +1,291 @@
+from flask import (
+    Blueprint,
+    flash,
+    redirect,
+    render_template,
+    request,
+    url_for,
+)
+from flask_login import (
+    current_user,
+    login_required,
+    login_user,
+    logout_user,
+)
+from flask_rq import get_queue
+
+from app import db
+from app.account.forms import (
+    ChangeEmailForm,
+    ChangePasswordForm,
+    CreatePasswordForm,
+    LoginForm,
+    RegistrationForm,
+    RequestResetPasswordForm,
+    ResetPasswordForm,
+)
+from app.email import send_email
+from app.models import User
+
+account = Blueprint('account', __name__)
+
+
+@account.route('/login', methods=['GET', 'POST'])
+def login():
+    """Log in an existing user."""
+    form = LoginForm()
+    if form.validate_on_submit():
+        user = User.query.filter_by(email=form.email.data).first()
+        if user is not None and user.password_hash is not None and \
+                user.verify_password(form.password.data):
+            login_user(user, form.remember_me.data)
+            flash('You are now logged in. Welcome back!', 'success')
+            return redirect(request.args.get('next') or url_for('main.index'))
+        else:
+            flash('Invalid email or password.', 'error')
+    return render_template('account/login.html', form=form)
+
+
+@account.route('/register', methods=['GET', 'POST'])
+def register():
+    """Register a new user, and send them a confirmation email."""
+    form = RegistrationForm()
+    if form.validate_on_submit():
+        user = User(
+            first_name=form.first_name.data,
+            last_name=form.last_name.data,
+            email=form.email.data,
+            password=form.password.data)
+        db.session.add(user)
+        db.session.commit()
+        token = user.generate_confirmation_token()
+        confirm_link = url_for('account.confirm', token=token, _external=True)
+        get_queue().enqueue(
+            send_email,
+            recipient=user.email,
+            subject='Confirm Your Account',
+            template='account/email/confirm',
+            user=user,
+            confirm_link=confirm_link)
+        flash('A confirmation link has been sent to {}.'.format(user.email),
+              'warning')
+        return redirect(url_for('main.index'))
+    return render_template('account/register.html', form=form)
```

Comment:
**Suggestion:** The register() function could be improved by separating the user creation logic from the routing logic. This would make the code more modular and easier to test.

```suggestion
def create_user(form):
    user = User(
        first_name=form.first_name.data,
        last_name=form.last_name.data,
        email=form.email.data,
        password=form.password.data)
    db.session.add(user)
    db.session.commit()
    return user

@account.route('/register', methods=['GET', 'POST'])
def register():
    """Register a new user, and send them a confirmation email."""
    form = RegistrationForm()
    if form.validate_on_submit():
        user = create_user(form)
        token = user.generate_confirmation_token()
        confirm_link = url_for('account.confirm', token=token, _external=True
```